docs/api/migration-guides/v2-primitives.mdx

```diff
@@ -0,0 +1,1298 @@
+- For V2 primitives, the `options` attribute is the `dataclass` Python type.  You can use the built-in `asdict` method to convert it to a dictionary. 
+
+{/*Verified EstimatorV2. 2/23 */}
+
+<Tabs>
+  <TabItem value="EstimatorV2" label="EstimatorV2">
+    ```python
+from dataclasses import asdict
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import EstimatorV2 as Estimator
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+estimator = Estimator(backend, options={"resilience_level": 2})
+
+# Setting options after primitive initialization
+# This uses auto complete.
+estimator.options.optimization_level = 1
+estimator.options.execution.shots = 4000
+# This does bulk update.
+estimator.options.update(shots=4000, optimization_level=1)
+
+# Print the dictionary format.
+# Server defaults are used for unset options.
+print(asdict(estimator.options))
+```
+  </TabItem>
+
+  <TabItem value="EstimatorV1" label="Estimator (V1)">
+    ```python
+from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Options
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+options = Options()
+# This uses auto complete.
+options.resilience_level = 2 
+estimator = Estimator(backend=backend, options=options)
+
+# Setting options after primitive initialization.
+# This does bulk update.
+estimator.set_options(shots=4000, optimization_level=1)
+```
+  </TabItem>
+</Tabs>
+
+<Tabs>
+  <TabItem value="SamplerV2" label="SamplerV2">
+  {/*from the boxnote */}
+    ```python
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit_ibm_runtime import SamplerV2 as Sampler
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+sampler = Sampler(backend)
+# Set optimization level to 0.
+sampler.options.optimization_level = 0
+# Turn on dynamical decoupling with sequence XpXm.
+sampler.options.dynamical_decoupling = "XpXm"
+
+print(f">> optimization level to use: {sampler.options.optimization_level}")
+print(f">> dynamical decoupling sequence to use: {sampler.options.dynamical_decoupling}")
+```    
+  </TabItem>
+
+  <TabItem value="SamplerV1" label="Sampler (V1)">
+    ```python
+from qiskit_ibm_runtime import QiskitRuntimeService, Sampler, Options
+
+service = QiskitRuntimeService()
+backend = service.least_busy(operational=True, simulator=False)
+
+# Setting options during primitive initialization
+options = Options()
+# This uses auto complete.
+options.resilience_level = 2 
+sampler = Sampler(backend=backend, options=options)
+
+# Setting options after primitive initialization.
+# This does bulk update.
+sampler.set_options(shots=2000, optimization_level=0)
+  ```  
+  </TabItem>
+</Tabs>
+
+### Advanced options
```

Comment:
What's the difference between `options` (from the previous section` and `advanced options`? Maybe we can say these are all available options for V2 primitives (and maybe hide the tables under twisties)